mysql/mysql.go

```diff
@@ -66,8 +69,13 @@ func ConfigFromURL(u *url.URL) (cfg *mysql.Config, err error) {
 	} else {
 		cfg = mysql.NewConfig()
 	}
-	cfg.Net = "tcp"
-	cfg.Addr = u.Host
+	if matches := netAddrRE.FindStringSubmatch(u.Host); len(matches) == 3 {
+		cfg.Net = matches[1]
+		cfg.Addr = matches[2]
+	} else {
+		cfg.Net = "tcp"
+		cfg.Addr = u.Host
+	}
```

Comment:
Unrelated to this PR, but maybe you can fix it anyway: There's a reference to "pq" down below in a docstring, which is PostGRES not MySQL, isn't that wrong?

Reply:
sure